Data/Sentinel2_baseline.qmd

```diff
@@ -0,0 +1,73 @@
+---
+format:
+  html:
+    code-fold: true
+execute:
+  echo: false
+  output: asis
+jupyter: python3
+
+---
+
+##### Additionally.....
```

Comment:
I suggest to change the wording from "Additionally....." to "Sentinel-2 L2A Baselines" (as Sentinel-2 L2A Baselines was the header of the document to be published.